cypress/e2e/smoke/dbot/checkViewDetails.cy.js

```diff
@@ -0,0 +1,185 @@
+import RunPanel from '../../../support/pageobjects/dbot/run_panel'
+import BotBuilder from '../../../support/pageobjects/dbot/bot_builder_page'
+import BotDashboard from '../../../support/pageobjects/dbot/bot_dashboard_page'
+
+describe('QATEST-4340 - View Details Button on Run Panel', () => {
+  const runPanel = new RunPanel()
+  const botBuilder = new BotBuilder()
+  const botDashboard = new BotDashboard()
+
+  it(`Checking View Details on Transaction tab for mobile'}`, () => {
+    cy.c_login({ user: 'dBot' })
+    cy.c_visitResponsive('/appstore/traders-hub', { size: 'mobile' })
+    cy.c_openDbotThub()
+    cy.findByTestId('close-icon', { timeout: 15000 }).click()
+    cy.c_skipTour()
+    botBuilder.openBotBuilderTab()
+    cy.c_skipTour()
+
+    // Checking if View Detail is disabled by default when there are no contracts
+    botDashboard.drawerToggleMobile.click()
+    runPanel.transactionsTab.click()
+    cy.findByRole('button', { name: 'View Detail' }).should('be.disabled')
+    botDashboard.drawerToggleMobile.click()
+
+    // Specifying asset and purchasing contracts
+    botBuilder.changeMarketOnBlocklyWorkspace(2, 'Volatility 100 (1s) Index')
+    cy.c_runBot()
+    cy.c_checkRunPanel()
+    cy.c_stopBot(6000)
+    runPanel.transactionsTab.click()
+
+    // Getting the details of contract from Transaction page and storing it
+    cy.findAllByTestId('dt_transactions_item_entry_spot')
+      .first()
+      .invoke('text')
+      .then((text) => {
+        const entrySpot = parseFloat(text)
+        cy.wrap(entrySpot).as('entrySpot')
+      })
+
+    cy.findAllByTestId('dt_transactions_item_exit_spot')
+      .first()
+      .invoke('text')
+      .then((text) => {
+        const exitSpot = parseFloat(text)
+        cy.wrap(exitSpot).as('exitSpot')
+      })
+
+    cy.findAllByTestId('dt_transactions_item_stake')
+      .first()
+      .invoke('text')
+      .then((text) => {
+        const stake = parseFloat(text)
+        cy.wrap(stake).as('stake')
+      })
+
+    cy.findAllByTestId('dt_transactions_item_profit')
+      .first()
+      .invoke('text')
+      .then((text) => {
+        const profitloss = parseFloat(text)
+        cy.wrap(profitloss).as('profitloss')
+      })
+
+    // Counting number of contracts in Transaction page
+    cy.findAllByTestId('dt_transactions_item')
+      .its('length')
+      .then((count) => {
+        cy.wrap(count).as('contractCount')
+      })
+
+    // Going to View detail page
+    cy.findByRole('button', { name: 'View Detail' })
+      .should('be.enabled')
+      .click()
+    cy.findByText('Transactions detailed summary').should('be.visible')
+
+    // Ensuring it has the same number of contracts in both Transaction and View details section
+    cy.findAllByTestId('dt_mobile_transaction_card')
+      .its('length')
+      .then((count) => {
+        cy.wrap(count).as('VcontractCount')
```

Comment:
For consistency, since we follow camelcase at most of the places:
```suggestion
        cy.wrap(count).as('vContractCount')
```